fixtures/coverall/tests/bindings/test_coverall.kts

```diff
@@ -181,13 +181,22 @@ Coveralls("test_complex_errors").use { coveralls ->
     } catch(e: ComplexException.PermissionDenied) {
         assert(e.reason == "Forbidden")
         assert(e.toString() == "uniffi.coverall.ComplexException\$PermissionDenied: reason=Forbidden") {
-            "Unexpected ComplexError.PermissionDenied.toString() value: ${e.toString()}"
+            "Unexpected ComplexException.PermissionDenied.toString() value: ${e.toString()}"
```

Comment:
This seems like an improvement but I don't understand how the Kotlin message changed without any changes to `bindings::kotlin`.

Reply:
This just changed the test's error message to match the rest of the code. Note that the same error is also called `ComplexException` a line above and also three lines above.